test/node/exports.test.ts

```diff
@@ -0,0 +1,82 @@
+import * as BSON from '../register-bson';
+import { sorted, byStrings } from './tools/utils';
+
+const EXPECTED_EXPORTS = [
+  // This is our added web indicator not a real export but a small exception for this test.
```

Comment:
Can you explain further why this is necessary?  I don't understand why __isWeb__ is added for our tests

Reply:
It is to identify when the export was created in an environment without Buffer (web-like, only TextEncoder/Decoder and btoa/atob globals). So we can change assertions based on that condition: for example, in the `error_test.js` file we can check instanceof on Node and `.name` prop in the web.